packages/react-ui/src/app/features/flows/lib/flows-hooks.ts

```diff
@@ -33,14 +31,11 @@ async function fetchFlows(name: string, limit: number, signal: AbortSignal) {
 }
 
 export const flowsHooks = {
-  useFlows: (request: Omit<ListFlowsRequest, 'projectId'>) => {
+  useFlows: (request: ListFlowsRequest) => {
     return useQuery({
-      queryKey: [QueryKeys.flows, authenticationSession.getProjectId()],
+      queryKey: [QueryKeys.flows],
```

Comment:
I'm not sure if this query key is correct, since we also use staleTime do you think we could have issues when we use search or pagination?